build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.6.3-8
+v1.6.3-9
```

Comment:
@lavalamp - test are failing because of this - if you agree that this is a good approach, I will build this image